packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Addresses/Btc/Wallets/template.tsx

```diff
@@ -1,27 +1,42 @@
+import { filter, take } from 'ramda'
+import { FormattedMessage } from 'react-intl'
+import { LinkContainer } from 'react-router-bootstrap'
+import React from 'react'
+import styled from 'styled-components'
+
 import {
   Banner,
-  IconButton,
+  Button,
   Link,
   Table,
   TableCell,
   TableHeader,
   TableRow,
   Text
 } from 'blockchain-info-components'
-import { filter, take } from 'ramda'
-import { FormattedMessage } from 'react-intl'
-import { LinkContainer } from 'react-router-bootstrap'
 import { SettingDescription, SettingHeader } from 'components/Setting'
 import media from 'services/ResponsiveService'
-import React from 'react'
-import styled from 'styled-components'
 import SwitchableDisplay from 'components/Display/SwitchableDisplay'
 
 const Wrapper = styled.section`
   box-sizing: border-box;
 `
-const BtcWalletsAddressesSettingHeader = styled(SettingHeader)`
+const TableStyled = styled(Table)`
+  > div:last-child {
+    border-bottom: none;
+  }
```

Comment:
we have exactly 4 times same const defined in different files, might be good to make one on common place and reuse

Reply:
will do